### PR TITLE
Adding jest and enzyme

### PR DESCRIPTION
**Pull request by Golodhros**
Adding jest and enzyme

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,18 +34,31 @@
...
     "webpack": "^3.6.0",
     "webpack-bundle-analyzer": "^2.9.0",
     "webpack-dashboard": "^1.0.0-6",
     "webpack-dev-server": "^2.8.2",
     "webpack-merge": "^4.1.0"
   },
   "dependencies": {
-    "britecharts": "^2.0.1",
-    "clean-webpack-plugin": "^0.1.16"
+    "britecharts": "^2.0.1"
```
Comment (rwholey-eb) [inline package.json]: we should probably have react and react dom here? any other ones we'll need?

File: package.json [hunk 2]
```diff
@@ -29,18 +53,31 @@
...
+  },
+  "jest": {
+    "moduleFileExtensions": [
```
Comment (rwholey-eb) [inline package.json]: i'm not sure you really need this right now right?

File: package.json (hunk 1 above)
Comment (Golodhros) [reply package.json]: Makes sense!
I think that's it for now...

File: package.json (hunk 2 above)
Comment (miglesiasEB) [reply package.json]: I am not sure!
In https://facebook.github.io/jest/docs/en/webpack.html#configuring-jest-to-find-our-files it says that this is equivalent to 'extensions' in webpack.
It seems to work without it!

**Pull request merged**